theoplayer/how-to-guides/01-ads/10-google-ima.md

```diff
@@ -192,7 +192,13 @@ SourceDescription sourceDescription = new SourceDescription.Builder()
 playerView.getPlayer().setSource(sourceDescription);
 ```
 
-### iOS/tvOS SDK and Legacy iOS/tvOS SDK (4.12.x)
+### iOS/tvOS SDK
+
+#### Current Version
+
+The documentation of the current version [can be found at this URL](https://github.com/THEOplayer/theoplayer-sdk-apple/tree/master/docsTHEOplayer-Integration-GoogleIMA/README.md).
+
+#### Legacy iOS/tvOS SDK (4.12.x)
```

Comment:
Just delete the legacy stuff. We have [our version 4.x docs](https://www.theoplayer.com/docs/theoplayer/v4/getting-started/sdks/ios/getting-started/) still up separately.

Reply:
done